src/index.html

```diff
@@ -1,17 +1,210 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post" name="personal-information">
+
+      <fieldset class="form-field personal-information">
```

Comment:
There are few mistakes here with the classes:
- here (and in many more elements) you are adding two separate classes which is unnecessary, classes are mostly used for css targeting and styling and you are not using this second class (`personal-information` or `surname` etc.) in any way
- class names like `input` are confusing especially when added to `div` elements
- apart from that, the name of the class should rather be connected with the context of the element or purpose, if you are using the same name as the some tag it's redundant (might as well just style it with the tag rather than class)
- I would rename `form-field` to something like `form-fields` (plural may be understood as label, input pair) and  `input` class to `form-field` (singular as one field of the form) -> these suggested class names are more generic (as they are re-used for many elements) and signify their context (they are used and belong to the form)